src/darwin/Framework/CHIP/MTRDeviceControllerParameters.h

```diff
@@ -79,8 +79,10 @@ MTR_AVAILABLE(ios(17.6), macos(14.6), watchos(10.6), tvos(17.6))
 
 /**
  * Sets the maximum subscriptions allowed for devices on Thread. This defaults to 3.
```

Comment:
Again:

> This makes it sound like we can only subscribe to at most 3 Thread devices. Needs to be rephrased to make it clear what this is really controlling.

Reply:
Ah I see what you mean. I'll add clarification.